include/groonga/column.h

```diff
@@ -57,8 +57,37 @@ typedef struct _grn_column_cache grn_column_cache;
  *
  * \since 3.1.1
  */
-#define GRN_COLUMN_NAME_ID_LEN       (sizeof(GRN_COLUMN_NAME_ID) - 1)
-#define GRN_COLUMN_NAME_KEY          "_key"
+#define GRN_COLUMN_NAME_ID_LEN (sizeof(GRN_COLUMN_NAME_ID) - 1)
+/**
+ * \brief Name of the pseudo column `_key`.
+ *
+ * The `_key` pseudo column represents the primary key value of a record in a
+ * Groonga table. It is defined only for tables that have a primary key. The
+ * primary key value is unique within the table and is immutable.
```

Comment:
なんと、`DAT_KEY`はキーをリネームできます。
https://groonga.org/ja/docs/reference/tables.html#characteristics

```suggestion
 * primary key value is unique within the table and is immutable except \ref GRN_TABLE_DAT_KEY. （一つの文に情報を詰め込むと読みにくい気はする。）
```

Reply:
なんと！なるほどです。ついでに文も分けてみます。

Reply:
fix: [c183efc](https://github.com/groonga/groonga/pull/2049/commits/c183efc2f2fe9aeac244eb619876e3f8c72ea15c)